adafruit_requests.py

```diff
@@ -672,7 +732,15 @@ def set_socket(sock, iface=None):
     sock.set_interface(iface)
 
 
-def request(method, url, data=None, json=None, headers=None, stream=False, timeout=1):
+def request(
+    method,
```

Comment:
What is the type of `method`?

Reply:
Also good catch!  I must missed this.  If I remember correctcly, `method` is a string defining the request method (GET, PUT, POST, etc)